src/FixerDefinition.php

```diff
@@ -38,7 +38,7 @@ public function __construct(
         array $codeSamples,
         $configurationDescription,
         array $defaultConfiguration = null,
-        $riskyDescription
+        $riskyDescription = null
```

Comment:
it will, it's ugly hack to make parameter nullable in PHP7 world, that's why they introduced `?type` in php 7.1